projects/canopy/src/lib/side-nav/side-nav.component.scss

```diff
@@ -0,0 +1,14 @@
+@import '../../styles/mixins';
+
+.lg-side-nav {
+  display: flex;
+  flex-direction: column-reverse;
```

Comment:
Interesting... never had a use for `column-reverse` before. I wonder if this still has implications for screen readers as they may still read the content out in the wrong order 🤷 . Any thought @elenagarrone ?

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/flex-direction#accessibility_concerns

Reply:
Will this cause an issue with assistive technology I wonder 🤔 . Perhaps not, as the menu will be read out first and then the content looking at the structure, but visually, it is placing the menu below

Reply:
Sorry, just saw this. it's shouldn't create issues in my opinion 🤔